scio-test/src/main/scala/com/spotify/scio/testing/JobTest.scala

```diff
@@ -115,13 +115,19 @@ object JobTest {
     def input[T: Coder](io: ScioIO[T], value: Iterable[T]): Builder =
       input(io, IterableInputSource(value))
 
+    def keyedInput[K: Coder, T: Coder](io: KeyedIO[K, T], value: Iterable[T]): Builder =
+      input(io, IterableInputSource(value.map(x => io.keyBy(x) -> x)))
+
     /**
      * Feed an input in the form of a `PTransform[PBegin, PCollection[T]]` to the pipeline being
      * tested. Note that `PTransform` inputs may not be supported for all `TestIO[T]` types.
      */
     def inputStream[T: Coder](io: ScioIO[T], stream: TestStream[T]): Builder =
       input(io, TestStreamInputSource(stream))
 
+    def keyedInputStream[K: Coder, T: Coder](io: KeyedIO[K, T], stream: Iterable[T]): Builder =
```

Comment:
Not sure if it is useful. What could be a hypothetical use case?

Reply:
As we have an API with `TestStream` for normal input, I just tried to keep consistency. 
It looks like `TestStream` is a more beam/java like way of giving some test data

Reply:
I get it. I forgot to replace with `TestStream` type